Test/TestDeconvolution.cs

```diff
@@ -47,7 +47,7 @@ public void TestDeconvolutionProteoformMultiChargeState(double selectedIonMz, in
 
             MsDataScan scan = myMsDataFile.GetAllScansList()[0];
 
-            List<IsotopicEnvelope> isolatedMasses = scan.GetIsolatedMassesAndCharges(spectrum, 1, 60, 4, 3).ToList();
+            List<MassSpectrometry.IsotopicEnvelope> isolatedMasses = scan.GetIsolatedMassesAndCharges(spectrum, 1, 60, 4, 3).ToList();
```

Comment:
this needs an assertion

Reply:
Changed test back to its original form, so there should be no need to add an assertion